automationspn/azuredeploy.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
The folder structure we discussed was:
```
/ (root)
/automation
/automation/functions
```
the `azuredeploy.json` and the `script.ps1` files should be in the `/automation/functions` folder. Structuring your files in an intuitive and logical way will help you better manage the source code when it grows and when you have more people and teams involved.